sympy/physics/vector/frame.py

```diff
@@ -697,7 +697,8 @@ def orientnew(self, newname, rot_type, amounts, rot_order='',
 
         """
 
-        newframe = self.__class__(newname, variables, indices, latexs)
+        newframe = self.__class__(newname, variables=variables,
+                                  indices=indices,latexs=latexs)
```

Comment:
There should be a space before and after '=' and after ','.

Reply:
I'm happy to make that change, but there are no whitespaces around equals signs in any of the method declarations or method calls anywhere else in this class (for example, line 824).  I believe PEP-8 recommends against white space for keyword operators (https://www.python.org/dev/peps/pep-0008/#other-recommendations).

Reply:
Ah! sorry I didn't notice that your right so just a space after ',' would be sufficient.